conf/backup_method

```diff
@@ -19,15 +19,16 @@ do_need_mount() {
 }
 
 do_backup() {
+    work_dir="$1"
+    name="$2"
+    size="$3"
+    description="$4"
     export BORG_PASSPHRASE
     export BORG_REPO
     export BORG_RSH
     export BORG_LOGGING_CONF
+    export BORG_FILES_CACHE_SUFFIX="${name}"
```

Comment:
Note to myself: probably worth to remove the old file cache in the upgrade script to save disk space.